fth/search.fth

```diff
@@ -0,0 +1,325 @@
+\ @(#) search.fth 5/06/20 0.3
+\ Search-Order wordset
+\
+\
+\
+\ Author: Hannu Vuolasaho
+\ Copyright 2015 3DO, Phil Burk, Larry Polansky, Devid Rosenboom
+\
+\ The pForth software code is dedicated to the public domain,

```

Comment:
License was changed to 0BSD
https://github.com/philburk/pforth/commit/1f99f95d6a7eecc05cae8fb357f9b7bf564c2725